src/02-crm/addresses.apib

```diff
@@ -0,0 +1,21 @@
+## Addressess [/levelTwoAreas]
+
+We provide a set of data which can be used to build addresses
+
+### levelTwoAreas.list [GET /levelTwoAreas.list]
+
+Gets all level two areas (which correspond to provinces, departments or states in most countries).
```

Comment:
PP: Get a list of level two areas .. (to be consistent with the other descriptions)